Workflow/Tests/StateMutationSinkTests.swift

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2022 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import ReactiveSwift
+import Workflow
+import XCTest
+
+final class StateMutationSinkTests: XCTestCase {
+    var output: Signal<Int, Never>!
+    var input: Signal<Int, Never>.Observer!
+
+    override func setUp() {
+        (output, input) = Signal<Int, Never>.pipe()
+    }
+
+    func test_initialValue() {
+        let host = WorkflowHost(workflow: TestWorkflow(value: 100, signal: output))
+        XCTAssertEqual(0, host.rendering.value)
+    }
+
+    func test_singleUpdate() {
+        let host = WorkflowHost(workflow: TestWorkflow(value: 100, signal: output))
+
+        let gotValueExpectation = expectation(description: "Got expected value")
```

Comment:
this doesn't really affect the semantics of the tests at all, but i _think_ the expectation machinery isn't strictly needed in this instance. the processing should happen synchronously as soon as the values are received.

Reply:
Would you prefer that I try removing it?

I just directly ported the test from `ios-register` and didn't dig into how I might be able to improve these tests much...